### DISCUSSION TITLE
Symbols Nerd Font fallback

### DISCUSSION THREAD

**Discussion by hansstimer** [zed-industries/zed#9150] [Feature Requests]
Apparently there is a way to use Symbols Nerd Font (mono) as a fallback font when the selected font is missing a glyph. This would be convenient so we can stop downloading patched fonts, or patching them ourselves. 

> Users can now install [Symbols Nerd Font (mono)](https://github.com/ryanoasis/nerd-fonts/releases/download/v2.2.2/NerdFontsSymbolsOnly.zip) from nerd fonts and use that as a fallback font if their terminal or application supports it.
> For Ex:
> 
> kitty terminal: Using symbols_map in kitty.conf as suggested in [kitty's FAQ](https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font)
> symbol_map <Unicode codepoints> Symbols Nerd Font
> 
> VS Code terminal: Using font fallback in settings.json
> "terminal.integrated.fontFamily": "'MonoLisa', 'Symbols Nerd Font'"

**Comment by simono** [comment]
For Zed on macOS you can do the following:
```shell
brew install font-symbols-only-nerd-font
```
And in Zed's `settings.json`:
```json
"buffer_font_fallbacks": ["Symbols Nerd Font"]
```

That will use Nerd Font as fallback for the Editor and Terminal in Zed.

**Comment by mil-ad** [comment]
is this meant to work in Linux? It's broken for me on Zed 0.214.7

**Reply by mskiptr** [reply]
You mean the `brew install …` command?

**Reply by mil-ad** [reply]
I meant the `buffer_font_fallbacks` option. Seems broken: #17254

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
